apis/core/bootstrap/shapes/cube-project.ts

```diff
@@ -210,6 +210,7 @@ ${shape('cube-project/search')} {
       ${sh.minCount} 1 ;
       ${sh.maxCount} 1 ;
       ${dash.editor} ${dash.InstancesSelectEditor} ;
+      ${sh1.orderBy} ( ${schema.name} ) ;
```

Comment:
just asking: I guess this change relies on the UI already supporting 'orderBy'

Reply:
Yes, added to shaperone, hence it was updated in `package.json`